api/client/cli.go

```diff
@@ -63,6 +63,14 @@ var funcMap = template.FuncMap{
 	},
 }
 
+func (cli *DockerCli) Out() io.Writer {
```

Comment:
Almost didn't need it  :-)  but then docker/docker.go need Err() at https://github.com/docker/docker/pull/13106/files#diff-b715a530d535012e418a564a3eafedb9R155  - and for consistency I added Out() too.